scheduler/test/cook/test/config.clj

```diff
@@ -97,3 +97,33 @@
                                [:config :kubernetes :controller-lock-num-shards]
                                256)]
       (is (thrown? ExceptionInfo (config-settings bad-config))))))
+
+(deftest test-guard-invalid-gpu-models
+  (testing "test"
```

Comment:
Maybe something more descriptive than "test"?

Reply:
changed to `(testing "empty valid-gpu-models"...`